src/webview_provider/docsEditPanel.ts

```diff
@@ -272,23 +278,23 @@ export class DocsEditViewPanel implements WebviewViewProvider {
                 } catch (exc) {
                   this.transmitError();
                   if (exc instanceof PythonException) {
-                    window.showErrorMessage(
-                      `An error occured while fetching metadata for ${modelName} from the database: ` +
-                        exc.exception.message,
-                    );
-                    this.telemetry.sendTelemetryError(
-                      "docsEditPanelLoadPythonError",
-                      exc,
+                    this.terminal.error(
+                      new CustomPythonException(
+                        "docsEditPanelLoadPythonError",
+                        `An error occured while fetching metadata for ${modelName} from the database`,
+                        exc,
+                      ),
+                      true,
```

Comment:
These error messages were written from the PoV of a error message following a user action. They are not necessarily written for logging (they might miss a lot of things like which project etc).

I think it is maybe better to keep these two concerns separate.